server/forms.py

```diff
@@ -458,3 +459,37 @@ class GithubSearchRecentForm(BaseForm):
                                 default="Academic Integrity - Please Delete This Repository")
     issue_body = TextAreaField('Issue Body (Optional)', validators=[validators.optional()],
                                description="The strings '{repo}' and '{author}' will be replace with the approriate value")
+
+##########
+# Canvas #
+##########
+
+# e.g. https://bcourses.berkeley.edu/courses/1234567
+CANVAS_COURSE_URL_REGEX = r'^https?://(([a-zA-Z0-9-]+\.)+[a-zA-Z0-9-]+)/courses/(\d+)'
```

Comment:
I was going to suggest constants.py - but it looks like this is only used in this file - so this is probably the right place for this.